src/tokens/TokenBalanceItem.tsx

```diff
@@ -34,7 +34,11 @@ export const TokenBalanceItem = ({
 
   const Content = (
     <View style={[styles.container, containerStyle]} testID="TokenBalanceItem">
-      <TokenIcon token={token} />
+      <TokenIcon
+        token={token}
+        showNetworkIcon={false}
+        viewStyle={{ borderColor: colors.black, borderWidth: 1, borderRadius: 20 }}
```

Comment:
In the designs the size of the token icon box is 40, so set borderRadius to 20 (40/2) here and it looks correct, but if there is a better way to do this I can change it.

Reply:
If you take a look at the figma, this actually is the new style for all TokenIcon's. Can you make this change in a way that it is now the default for TokenIcon accross the app?

Reply:
added in [8f9a5d5](https://github.com/mobilestack-xyz/mobilestack-mento/pull/9/commits/8f9a5d5e6d0e829106ad8b4dc363463a815c32e1)